src/RegisterErc20Tokens/Program.cs

```diff
@@ -0,0 +1,437 @@
+                            if (oldToken != null)
+                            {
+                                oldToken.AssetId = null;
+                                _assetsService.Erc20TokenUpdate(oldToken);
+                            }
+
+                            asset.BlockChainAssetId = tokenAddress;
+                            erc20Token = erc20Token ?? _assetsService.Erc20TokenGetByAddress(tokenAddress);
+                            erc20Token.AssetId = assetId;
+                            _assetsService.AssetUpdate(asset);
+                            _assetsService.Erc20TokenUpdate(erc20Token);
+                        }
+                        else
+                        {
+                            assetIds.Add(assetId);
+                        }
+                    }
+                }
+                catch (Exception e)
+                {
+                    operationResult = e.ToOperationResult();
+                    operationSucceeded = false;
+                }
+
+                OutputOperationResult(contract.Address, operationResult, operationSucceeded);
+            });
+
+            return assetIds;
+        }
+
+        private static void EnableAssets(IEnumerable<string> assetIds)
+        {
+            ResetOutputCounter();
+
+            Console.ResetColor();
+            Console.WriteLine("Enabling assets...");
+
+            Parallel.ForEach(assetIds, assetId =>
+            {
+                var operationResult = "";
+                var operationSucceeded = true;
+
+                try
+                {
+                    _assetsService.AssetEnable(assetId);
+
+                    operationResult = "asset enabled";
+                }
+                catch (Exception e)
+                {
+                    operationResult = e.ToOperationResult();
+                    operationSucceeded = false;
+                }
+
+                OutputOperationResult(assetId, operationResult, operationSucceeded);
+            });
+        }
+
+        private static bool TryProcessArgs(string[] args, out (Uri AssetsServiceUri, Uri Web3Uri, string SpecificationPath) processedArgs)
+        {
+            processedArgs = (null, null, null);
+
+            try
+            {
+                processedArgs.AssetsServiceUri = new Uri(args[0], UriKind.Absolute);
+            }
+            catch (Exception)
+            {
+                Console.WriteLine("First argument should be a valid absolute uri string.");
+
+                return false;
+            }
+
+            try
+            {
+                processedArgs.Web3Uri = new Uri(args[1], UriKind.Absolute);
+            }
+            catch (Exception)
+            {
+                Console.WriteLine("Second argument should be a valid absolute uri string.");
+
+                return false;
+            }
+
+            try
+            {
+                if (!File.Exists(args[2]))
+                {
+                    throw new Exception();
```

Comment:
Add description to the exception and probably write to console and return false, as for another parameters